docs/package.json

```diff
@@ -18,6 +18,7 @@
     "@nuxtjs/plausible": "^1.0.0",
     "@types/node": "^20.12.12",
     "eslint": "^9.2.0",
-    "nuxt": "^3.11.2"
+    "nuxt": "^3.11.2",
+    "pinceau": "0.18.9"
```

Comment:
`pinceau` is imported but not explicitly declared as a devDependency; resolves an import error